resources/website-l10n.xsl

```diff
@@ -55,50 +55,50 @@
 
     <l:l10n language="zh">
       <l:context name="title">
-        <l:template name="part"    text="%t"/>
-        <l:template name="chapter" text="%t"/>
-        <l:template name="section" text="%t"/>
+        <l:template name="part"    text="部分"/>
```

Comment:
Sorry should have explained.  `%t` is replaced by the title of the element being linked to.  You probably want to leave this as it is.